src/test/suite/tokenizer.test.ts

```diff
@@ -60,13 +67,53 @@ const tests: TestCase[] = [
   ["aåäöb", ["aåäöb"]],
 ];
 
+const cssDialectTokenizerTests: LanguageTokenizerTests[SupportedLanguageId] = {
+  tests: [
+    ["min-height", ["min-height"]],
+    ["-webkit-font-smoothing", ["-webkit-font-smoothing"]],
+    ["(min-width: 400px)", ["(", "min-width", ":", "400px", ")"]],
+    ["prefers-reduced-motion", ["prefers-reduced-motion"]],
+  ],
+  // Leave kebab and dashes to css language specific tests.
+  exclusionPredicate: (input: string) => !!input.match("-"),
+};
+
+const languageTokenizerTests: LanguageTokenizerTests = {
```

Comment:
Then here you just use 
```suggestion
const languageTokenizerTests: Partial<Record<SupportedLanguageId, LanguageTokenizerTests>> = {
```